### ISSUE TITLE
Cordova variables missing during another platform build if a Cordova plugin is installed from GitHub

### ISSUE THREAD

**Issue by nelsyeung** [meteor/meteor#11127]
## Reproduction details

Terminal commands:

```sh
meteor create todos
cd todos
meteor add accounts-google
meteor add "cordova:cordova-plugin-statusbar@https://github.com/breautek/cordova-plugin-statusbar.git#430e7df32c0937d46e3a596b37ecbd50a42cb9c3"
```

Add the following to `mobile-config.js`:

```javascript
App.setPreference('android-minSdkVersion', '21');
App.configurePlugin('cordova-plugin-googleplus', { REVERSED_CLIENT_ID: 'foo' });
```

Terminal commands:

```sh
meteor add-platform ios
meteor add-platform android
meteor run ios-device  # This will run fine
meteor run android-device
```

Errors:

<details>
<summary>

```
While removing plugins
cordova-plugin-googleplus,cordova-plugin-inappbrowser,cordova-plugin-meteor-webapp,cordova-plugin-splashscreen,cordova-plugin-statusbar,cordova-plugin-whitelist,cordova-plugin-wkwebview-engine
from Cordova project:
Error: Variable(s) missing: REVERSED_CLIENT_ID
at Object.mergeVariables
(/Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/cordova-lib/src/plugman/variable-merge.js:58:15)
at runUninstallPlatform
```

</summary>

```sh
=> Started proxy.
=> Errors executing Cordova commands:

   While removing plugins
   cordova-plugin-googleplus,cordova-plugin-inappbrowser,cordova-plugin-meteor-webapp,cordova-plugin-splashscreen,cordova-plugin-statusbar,cordova-plugin-whitelist,cordova-plugin-wkwebview-engine
   from Cordova project:
   Error: Variable(s) missing: REVERSED_CLIENT_ID
   at Object.mergeVariables
   (/Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/cordova-lib/src/plugman/variable-merge.js:58:15)
   at runUninstallPlatform
   (/Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/cordova-lib/src/plugman/uninstall.js:245:35)
   at
   /Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/cordova-lib/src/plugman/uninstall.js:90:16
   at
   /Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/meteor-promise/fiber_pool.js:43:40
   => awaited here:
   at Function.Promise.await
   (/Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:56:12)
   at CordovaProject.runCommands (/tools/cordova/project.js:842:22)
   at CordovaProject.removePlugins (/tools/cordova/project.js:573:10)
   at /tools/cordova/project.js:702:14
   at Object.enterJob (/tools/utils/buildmessage.js:388:12)
   at CordovaProject.ensurePluginsAreSynchronized (/tools/cordova/project.js:584:18)
   at CordovaProject.prepareFromAppBundle (/tools/cordova/project.js:242:10)
   at /tools/cordova/runner.js:78:27
   at Object.enterJob (/tools/utils/buildmessage.js:388:12)
   at CordovaRunner.prepareProject (/tools/cordova/runner.js:77:18)
   at /tools/runners/run-app.js:661:25
   at Object.capture (/tools/utils/buildmessage.js:283:5)
   at AppRunner._runOnce (/tools/runners/run-app.js:660:39)
   at AppRunner._fiber (/tools/runners/run-app.js:927:28)
   at /tools/runners/run-app.js:401:12

/Users/nelsyeung/.meteor/packages/meteor-tool/.1.10.2.6hsx6.cly95j++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:218
      throw error;
      ^
ExitWithCode { code: 1 }
```

</details>

## Workaround

Reset before the second build `meteor reset`.

## Notes

* Running iOS or Android first doesn't matter, it'll always break during the second build.
* Installing without GitHub is fine (i.e., running `meteor add cordova:cordova-plugin-statusbar@2.4.3` works fine).
* **OS**: macOS 10.15.5
* **Meteor**: 1.10.2
* **Node**: 14.5.0

**Comment by crapthings** [comment]
i apply this to local meteor 1.9.3 cordova build tool, it works.
https://github.com/meteor/meteor/tree/variable-missing-removing-cordova-plugin

**Comment by filipenevola** [comment]
Hi @nelsyeung, this was fixed on Meteor 1.11
